plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/query/action.py

```diff
@@ -40,21 +49,23 @@ def run(self, params={}):
             self.logger.error(f"InsightIDR response: {response}")
             raise PluginException(
                 cause="The response from InsightIDR was not in the correct format.",
-                assistance="Contact support for help. See log for more details.",
+                assistance="Contact support for help. See log for more details."
             )
 
         try:
             result_response = []
             events = result.get("events", [])
             if events:
                 for event in events:
-                    event["message"] = json.loads(event["message"].replace("\n", "\\n"))
+                    print(json.loads(event["message"].replace("\n", "\\n")))
+                    # event["message"] = json.loads(event["message"].replace("\n", "\\n"))
```

Comment:
I think we'd want to take out the print statement & comment before final version